convert2rhel/utils.py

```diff
@@ -125,6 +125,123 @@ def restart_system():
         loggerinst.warning("In order to boot the RHEL kernel, restart of the system is needed.")
 
 
+def _normalize_subprocess_output(output, skip_stdout_decode):
```

Comment:
This function is intended to be used in the `run_subprocess` function too in the future, but that could be a separate PR.

Reply:
Note: I see that the tests/integration/conftest.py could benefit from the `skip_stdout_decode` logic too.